src/main/java/com/neighborhoodAssurance/NAController.java

```diff
@@ -28,4 +34,10 @@ public ModelAndView home() {
 	public String readJSON() {
 		return "home";
 	}
+
+	@RequestMapping(value="/searchResults", method=RequestMethod.GET)
+	public String doSearch(@RequestParam String searchTerm, @RequestParam int submit) {
+		logger.info("Request successful we got: " + searchTerm);
+		return "home";
+	}
 }
```

Comment:
Once I got your site building appropriately I noticed that you commented out a form. I removed the comments and hooked it up so it works. Currently it only logs out the message but you get the idea. Feel free to use the searchTerm param however you see fit in your searching function. You can obviously expand on this quite a bit.